tests/benchmark/run.sh

```diff
@@ -76,7 +76,7 @@ echo - Output: $(get_abs_filename $results_file)
           cat $results_file
 
           if [[ -z "${GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
```

Comment:
@tatiana question: is there a recommended way to get credentials? since we are getting it from the env variable only?

Reply:
Yeah I had a lot of trouble with this, google doesn't make this easy so it would be great to have some basic docs on how to set it all up.